notebooks/Notebook1.md

```diff
@@ -9,6 +9,16 @@ Did it help you make a design decision? Was it helpful in addressing the most pr
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+I received feedback and ideas about some of the new operators I could introduce, and maybe even an "if" statement structure. I had thought about a lot of these before, but it was still helpful to talk about it and explain it to my critique partners. Just talking and explaining helps clarify, and maybe even get new ideas from both sides. 
+
+Other people's DSL implementation ideas were also helpful. On Monday we spoke about using excel spreedsheets, APIs, regex, and some other interesting implementation ideas which, while I don't think I will be using just yet - but it helped me branch out my line of thinking. And some of these ideas may be important in the future, if and when I run into issues and barriers with my own implementation.
+
+Was it helpful in addressing the most pressing issues in your project?
+
+I still have not run into pressing issues, since it is really only the first week of the project - but I look forward to future critique sessions. 
+
+A couple ideas of mine are still abstract - and I was hoping to create a live list of operators and features I would like to add. This is would help grounding the project, and also help future critique sessions!
```

Comment:
I agree. This would make the language's capabilities clearer and also help you sort out which features you want to add. You could also rank each operator and feature by importance, so you can know which to prioritize when designing it.